CHANGELOG.md

```diff
@@ -4,6 +4,95 @@
 
 # JupyterLab Changelog
 
+## v4.4
+
+JupyterLab 4.4 includes a number of new features (described below), bug fixes, and enhancements.
+This release is compatible with extensions supporting JupyterLab 4.0.
+Extension authors are recommended to consult the [Extension Migration Guide](https://jupyterlab.readthedocs.io/en/latest/extension/extension_migration.html#jupyterlab-4-3-to-4-4) which lists deprecations and changes to the public API.
+
+### Code console improvements
+
+The code console prompt can now be positioned on top, left, or right side of the console,
+in addition to the default bottom placement. The placement is persisted in the Prompt Cell
+Position setting.
+
+The console toolbar now includes buttons to:
+- execute code,
+- restart the kernel,
+- clear the cells,
+- switch the kernel.
+
+Additional settings for console behavior were added:
+- Clear Code Content on Execute (enabled by default) - when disabled,
+  the code submitted for execution remains in the code editor, allowing for further edits
+- Hide Code Input - when enabled shows only the output of the execution in the console
+- Clear Cells on Execute - when enabled shows only last cell
+
+Toggling all the three new settings transforms the console into an interactive editor
```

Comment:
```suggestion
Enabling all three of these new settings transforms the console into an interactive editor
```

Reply:
"Clear Code Content on Execute" is enabled by default and needs to be disabled, right? Hence "toggling"

Reply:
Ah right,, leaving as is then!